File: fibonacci/js/code.js

function fibonacci(n) {
  if (n == 0) return 0;
  if (n == 1) return 1;
  return fibonacci(n-1) + fibonacci(n-2);
}

var u = Number(process.argv[2]);
var r = 0;
for (let i = 1; i < u; i++) {
  r += fibonacci(i);
}
console.log(r);

File: fibonacci/c/code.c
#include "stdio.h"
#include "stdlib.h"
#include "stdint.h"

int32_t fibonacci(int32_t n) {
  if (n == 0) return 0;
  if (n == 1) return 1;
  return fibonacci(n-1) + fibonacci(n-2);
}

int main (int argc, char** argv) {
  int32_t u = atoi(argv[1]);
  int32_t r = 0;
  for (int32_t i = 1; i < u; i++) {
    r += fibonacci(i);
  }
  printf("%d\n", r);
}
